packages/ui-text-input/src/TextInput/index.tsx

```diff
@@ -65,7 +65,7 @@ type Props = {
   onFocus?: (...args: any[]) => any
   makeStyles?: (...args: any[]) => any
   styles?: any
-}
+} & InputHTMLAttributes<Props>
```

Comment:
I think we will need this on every Component, that we add all available props that can be added on that element (eg. aria tags)